### PR TITLE
Upgrade dockerfile-demo-DB and path ci settings-demo.ini

### PR DESCRIPTION
**Pull request by esloch**
- Foi criado um path para o arquivo settings-demo.ini para evita que os dados fique expostos.
- Foi feito um update no dockerfile e renomeado para dockerfile_demoDB. (Para auxiliar na criação do ambiente de desenvolvimento).

### CODE REVIEW

File: dockerfile-demo-DB [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+RUN cd ~ && \
+ bash Miniconda.sh -b -p /home/dengue/miniconda && \
+ rm Miniconda.sh
+
+#GIT clone alias DEVELOP
+RUN git clone -b develop https://github.com/AlertaDengue/AlertaDengue.git
+
+ENV PATH="/home/dengue/miniconda/bin:${PATH}"
+
+RUN /home/dengue/AlertaDengue/AlertaDengue/environment.yml /tmp/environment.yml
```
Comment (xmnlab) [inline dockerfile-demo-DB]: these 2 RUN instruction (this line and below, about environment.yml and requirements-dev) seems to be wrong, probably you are trying to use ADD .. but also it is unnecessary  ... as you are cloning the repo inside the image .. you don't need to add any file from host.

**Comment by xmnlab** [comment]
for some reason you don't have the folder .circleci in your code (https://github.com/AlertaDengue/AlertaDengue/tree/1aaddde2d38f09ff3b0dfc6072179be6741cf2a8)

check if the commit before your commit here https://github.com/AlertaDengue/AlertaDengue/commits/1aaddde2d38f09ff3b0dfc6072179be6741cf2a8 is `216472f`

it seems your branch is missing the last 20 commits ... so probably you should rebase your code on top of upstream/develop

File: dockerfile-demo-DB (hunk 1 above)
Comment (esloch) [reply dockerfile-demo-DB]: ! Yes, in fact the two lines of RUN envirioment is residue of when we were doing the ADD of the local envirioment files.
I'll make a new PR ...

**Comment by fccoelho** [comment]
@esloch have you performed the rebase suggested by @xmnlab ? If so, maybe we can merge this PR.

**Comment by xmnlab** [comment]
@fccoelho it seems it is Ok to be merged :)

**Pull request merged**